components/track/TrackList.vue

```diff
@@ -13,9 +13,22 @@ withDefaults(
     skeletonCount: 5,
   }
 );
+
 const { setCurrentTrack, addTrackToQueue } = inject(MediaPlaylistInjectionKey)!;
 const showDropDownForTrack: Ref<null | string> = ref(null);
 
+const isTrackTypeKnown = () => {
+  const firstType = props.tracks?.[0]?.subtype;
+  console.log("first type", firstType);
```

Comment:
To me this doesn't look like its very valuable if this function is working as expected. Please remove before merging or make it more intuitive to read and relate.

Reply:
I guess you mean the console.log that I overlooked.